<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_34">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>模糊知识知识库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="FuzzyKtable_3">
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>15</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>条件变量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模糊概念A</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>结论变量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>模糊概念B</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>知识可信度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>条件可信度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>更新时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>更新人</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,0">
     <item>
      <spacer name="verticalSpacer_21">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ID_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_22">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>条件变量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConditionalVar_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_23">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>条件变量类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConditionalVarType_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_24">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>模糊概念A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FuzzyConA_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_25">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_47" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_45">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>隶属函数A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FuzzyFuncA_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_43">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>结论变量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConclusionVar_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_44">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>结论变量类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConclusionVarType_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_26">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>模糊概念B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FuzzyConB_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_27">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_48" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_46">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>隶属函数B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FuzzyFuncB_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_45">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>条件可信度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConditionalCre_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_28">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>知识可信度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="KnowledgeCre_3"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_29">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_32" stretch="1,2">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>更新人</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="UpdatePeople_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_30">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_33">
       <item>
        <widget class="QPushButton" name="InsertButton_3">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DeleteButton_3">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="UpdateButton_3">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="QueryButton_3">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearButton_3">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
